nautobot/extras/forms/mixins.py

```diff
@@ -538,6 +538,7 @@ def _append_relationships_side(self, relationships, initial_side):
                 # No need to process it a second time!
                 continue
             self.fields[field_name] = relationship.to_form_field(side=side)
+            self.fields[field_name].empty_label = None
```

Comment:
This fixes the error " "" is not a valid UUID" we saw during the demo.

Reply:
Any chance this should be added directly in the `to_form_field` method? Or is this specific to the filterform case and not applicable to other uses of relationships in forms?

Reply:
I think this addresses the issue that we use `ModelMultipleChoiceFilter` to filter `ModelChoiceField` (one to many_source field) and the field is empty. Disabling the `empty_label `prevents the field from sending an empty "" to the querystring in the url when the field is empty. In my experience, it does not change the way other use cases of relationships behave so I think either way is fine.